channels/us.m3u

```diff
@@ -3660,3 +3660,5 @@ https://rockentertainment-zoomoo-1.samsung.wurl.com/manifest/playlist.m3u8
 http://188.40.68.167/russia/disney/playlist.m3u8
 #EXTINF:-1 tvg-id="MeiGuoQieDaDianShi.us" tvg-name="美国切达电视" tvg-country="US" tvg-language="" tvg-logo="" group-title="",美国切达电视 (1080p)
 https://live.chdrstatic.com/cheddar/primary/4.m3u8
+#EXTINF:-1 tvg-id="WUVP.us" tvg-name="WUVP-DT1" tvg-country="US" tvg-language="US" tvg-logo="https://static.locastnet.org/logo/504/1613144123818_UNI.png" group-title="",Univision Philadelphia (WUVP)
```

Comment:
This stream is `[Geo-blocked]`, please add this label to the title.
![image](https://user-images.githubusercontent.com/7253922/121783323-99d08c80-cbb6-11eb-8bb0-86ed6e3ca5af.png)

Reply:
Fixed try again shouldn't be blocked now.

Reply:
![vlcsnap-2021-06-12-19h01m47s721](https://user-images.githubusercontent.com/30985701/121783746-a81baa00-cbb0-11eb-90a1-38c049987258.png)

Works for me.

Reply:
Now it works!